spec/models/account_cross_search_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe AccountCrossSearch, type: :model do
+  it do
+    is_expected.to belong_to(:full_account)
+      .class_name('Account')
+
+    is_expected.to belong_to(:search_account)
+      .class_name('Account')
```

Comment:
Can these be on a single line to make it clearer what they are doing?

```
is_expected.to belong_to(:search_account).class_name('Account')
```

Reply:
Thanks done.